python/lsst/ts/watcher/rules/generic_boolean.py

```diff
@@ -0,0 +1,162 @@
+                callback_names=[callback_name],
+                poll_names=[],
+            )
+        ]
+        super().__init__(
+            config=config,
+            name=f"{self.remote_name}:{self.remote_index}.{rule_name}",
+            remote_info_list=remote_info_list,
+            log=log,
+        )
+
+    @classmethod
+    def get_schema(cls):
+        enum_str = ", ".join(
+            f"{severity.name}"
+            for severity in AlarmSeverity
+            if severity is not AlarmSeverity.NONE
+        )
+        schema_yaml = f"""
+$schema: http://json-schema.org/draft-07/schema#
+description: Configuration for GenericBoolean rule.
+type: object
+properties:
+  remote_name:
+    description: The name of the Remote.
+    type: string
+  remote_index:
+    description: The index of the Remote.
+    type: number
+    default: 0
+  callback_name:
+    description: The name of the callback, i.e. the SAL topic.
+    type: string
+  alarm_items:
+    description: Alarm items in the SAL topic.
+    type: array
+    minItems: 1
+    items:
+      description: The name of the topic item.
+      type: string
+  severity:
+    description: Alarm severity.
+    type: string
+    enum: [{enum_str}]
+required:
+  - remote_name
+  - remote_index
+  - callback_name
+  - alarm_items
+  - severity
+additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def compute_alarm_severity(
+        self, data: salobj.BaseMsgType, **kwargs: typing.Any
+    ) -> AlarmSeverityReasonType:
+        """Compute and set alarm severity and reason.
+
+        Parameters
+        ----------
+        data : `salobj.BaseMsgType`
+              Message from the topic described by topic_callback.
+        **kwargs : `dict` [`str`, `typing.Any`]
+            Keyword arguments. If triggered by `TopicCallback` calling
+            `update_alarm_severity`, the arguments will be as follows:
+
+            * topic_callback : `TopicCallback`
+              Topic callback wrapper.
+
+        Returns
+        -------
+        None, if no change or unknown, or a tuple of two values:
+
+        severity: `lsst.ts.xml.enums.Watcher.AlarmSeverity`
+            The new alarm severity.
+        reason : `str`
+            Detailed reason for the severity, e.g. a string describing
+            what value is out of range, and what the range is.
+            If ``severity`` is ``NONE`` then this value is ignored (but still
+            required) and the old reason is retained until the alarm is reset
+            to ``nominal`` state.
+
+        Notes
+        -----
+        You may return `NoneNoReason` if the alarm state is ``NONE``.
+        """
+        severity, reason = NoneNoReason
+
+        alarm_reasons = []
+        for alarm_item in self.config.alarm_items:
+            item_value = getattr(data, alarm_item, False)
+            if item_value is True:
+                severity = AlarmSeverity.CRITICAL
```

Comment:
You included the severity in the configuration but here you are hard-coding it to `CRITICAL`.